content/posts/jellyfin-10-6-0.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Jellyfin Release - v10.6.0"
+subtitle: "Party together while social distancing"
+author: "Julien Machiels"
+githubusername: "MrTimscampi"
+date: 2020-07-13T11:42:00+01:00
+justify: "center"
+---
+
+After months of work, here comes a behemoth of a release, with over 30 major improvements.
```

Comment:
What about
After months of work, here comes the next behemoth of a release

we keep doing behemoth releases x)